pool/cache.go

```diff
@@ -21,15 +27,31 @@ func NewCache() (*SessionCache, error) {
 }
 
 func (c *SessionCache) Get(key string) *session.Token {
-	tokenRaw, ok := c.cache.Get(key)
+	valueRaw, ok := c.cache.Get(key)
 	if !ok {
 		return nil
 	}
-	return tokenRaw.(*session.Token)
+
+	value := valueRaw.(*cacheValue)
+	value.atime = time.Now()
+
+	return value.token
+}
+
+func (c *SessionCache) GetAccessTime(key string) (time.Time, bool) {
+	valueRaw, ok := c.cache.Get(key)
```

Comment:
Maybe `Get` should be replaced by `Peek`? As I understand, we don't need to update 'recenly used`ness of the value.

Reply:
Fixed.